website/src/content/blog/biome-v1-7.md

```diff
@@ -0,0 +1,306 @@
+---
+title: Biome v1.7
+description: Migrate from Prettier and ESLint with one command!
+summary: |
+  This new version provides an easy path to migrate from ESLint and Prettier.
+  It also introduces machine-readable reports for the formatter and the linter, new linter rules, and many fixes.
+authors:
+  - conaclos
+  - team
+pubDate: 2024-04-15
+coverImage:
+  lightSrc: "@/assets/blog/roadmap-2024/banner-light.png"
+  darkSrc: "@/assets/blog/roadmap-2024/banner-dark.png"
+  alt: The brand of the project. It says "Biome, toolchain of the web"
+socialImage: "@/assets/social-logo.png"
+---
+
+Today we’re excited to announce the release of _Biome v1.7_!
+
+This new version provides an easy path to migrate from ESLint and Prettier.
+It also introduces experimental machine-readable reports for the formatter and the linter, new linter rules, and many fixes.
+
+If you’re not familiar with _Biome_ yet, it is a fast formatter and linter for JavaScript, TypeScript, JSX, and JSON that scores [97% compatibility with Prettier](https://console.algora.io/challenges/prettier) and provides [more than 200 linter rules](/linter/rules/).
```

Comment:
I'm not really sure about this paragraph. The focus of the blog post is the presentation of the new features. Should we remove it?